snappi_ixnetwork/trafficitem.py

```diff
@@ -298,6 +300,15 @@ class TrafficItem(CustomField):
         "order": ["length", "data"],
     }
 
+    _VXLAN = {
+        "flags": "vxlan.header.flags",
+        "reserved0": "vxlan.header.reserved",
+        "vni": "vxlan.header.vni",
+        "reserved1": "vxlan.header.reserved8",
+        "order": ["flags", "reserved0", "vni", "reserved1"],
+        "convert_int_to_hex": ["flags"],
```

Comment:
Looks like "reserved0" and "reserved1" also required "convert_int_to_hex"

Reply:
thanks for the correction